iib/workers/config.py

```diff
@@ -218,6 +218,19 @@ class DevelopmentConfig(Config):
         str
     ] = '/var/lib/requests/recursive_related_bundles'
     iib_dogpile_backend: str = 'dogpile.cache.memcached'
+    iib_ocp_opm_mapping: dict = {
```

Comment:
I am missing some config validation in which I would like  to check if all these omp version really exists in file system. If not raise an error.

Reply:
Updated